streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -1831,7 +1848,7 @@ public void shouldReturnOffsetsForRepartitionTopicsForPurging() {
             taskId,
             config,
             stateManager,
-            streamsMetrics,
+            this.mockStreamsMetrics,
```

Comment:
nit: We do not use `this` if it is not needed:
 ```suggestion
            mockStreamsMetrics,
```
Could you please adapt the remainder of this class accordingly?